src/main/java/baseball/model/PlayerChoice.java

```diff
@@ -0,0 +1,21 @@
+package baseball.model;
+
+public class PlayerChoice {
+    private int value;
+
+    public PlayerChoice(String input) {
+        validateChoice(input);
+        value = Integer.parseInt(input);
+    }
+
+    private void validateChoice(String input) {
+        int choice = Integer.parseInt(input);
```

Comment:
생성자에서의 parseInt이랑 중복되는 같아요. 인자로 input을 받는 대신 파싱된 value를 이용해서 validate 함수를 만들어도 좋을 거 같아요!

Reply:
> 생성자에서의 parseInt이랑 중복되는 같아요. 인자로 input을 받는 대신 파싱된 value를 이용해서 validate 함수를 만들어도 좋을 거 같아요!

오 그렇네요!! 감사합니다!! 
뭔가 이상해서 추가로 확인해보니... 문자열을 입력받았을 때 예외처리를 안했네요ㅠㅠㅠ
아래처럼 바꿨습니다. 

```suggestion
public PlayerChoice(String input) {
    int inputToIntValue = validateNumeric(input);
    validateValue(inputToIntValue);
    value = inputToIntValue;
}

private int validateNumeric(String input) {
       int intInputValue;
       try {
       intInputValue  = Integer.parseInt(input);
       } catch (NumberFormatException e) {
       throw new IllegalArgumentException("[ERROR] 1 또는 2를 입력해주세요.");
       } 
       return intInputValue;  
}
      
private void validateValue(int input) {
      if ((input != 1) && (input != 2)) {
           throw new IllegalArgumentException("[ERROR] 1 또는 2를 입력해주세요.");
       }   
 }
```